extensions/wikia/UserLogin/js/MarketingOptIn.js

```diff
@@ -12,21 +12,21 @@ define('usersignup.marketingOptIn', ['wikia.geo'], function (geo) {
 		var $optInField = wikiaForm.inputs.wpMarketingOptIn,
 			$optInGroup = wikiaForm.getInputGroup('wpMarketingOptIn'),
 			isEurope = geo.getContinentCode() === 'EU',
-			isCanada = geo.getCountryCode() === 'CA';
+			isCanada = geo.getCountryCode() === 'CA',
+			isJapan = geo.getCountryCode() === 'JP';
```

Comment:
I agree the way I did this isn't very sophisticated and wouldn't stand up against adding a bunch of countries and a bunch of rules. I could have mentioned this in the PR description, but we're going to revisit (and ultimately replace) this code with https://wikia-inc.atlassian.net/browse/SOC-560, so I didn't want to spend too much effort optimizing just yet.